### ISSUE TITLE
[NEW ICON] openSUSE, Fedora, Dreamweaver, Joomla, Drupal

### ISSUE THREAD

**Issue by TommyHyperTexter** [tandpfun/skill-icons#941]
### Skill Name

openSUSE, Fedora, Dreamweaver, Joomla, Drupal

### Why?

Popular Linux distributions.
Popular web development software.
Popular content management systems.

### Reference Image

![openSUSE](https://github.com/tandpfun/skill-icons/assets/109879398/b7a2bd64-d87c-4dee-8432-a67486b67aa6)
![Fedora](https://github.com/tandpfun/skill-icons/assets/109879398/40ad6d63-6a5f-47af-8289-7104358f3c51)
![Dreamweaver](https://github.com/tandpfun/skill-icons/assets/109879398/cbd25bde-82e9-4cac-bf7f-61ad4584ccd8)
![Joomla](https://github.com/tandpfun/skill-icons/assets/109879398/4a1ac97f-1d6e-4af5-93bb-861bc9419a6a)
![Drupal](https://github.com/tandpfun/skill-icons/assets/109879398/2071e2e7-bb7e-43fd-a838-732b4b68edd6)

**Comment by Jeesson** [comment]
Adobe Photoshop and Illustrator are available right now…

**Comment by TommyHyperTexter** [comment]
> Adobe Photoshop and Illustrator are available right now…

You're right, my mistake. I didn't realize the names were written in short form, therefore when I searched the page for "Photoshop" and "Illustrator" they came up as "Phrase not found".

**Comment by Jeesson** [comment]
It's okay, you can edit your suggestion at any time, by the way 👍🏻
UPD: Oh, by the way, I just wanted to let you know that I made my suggestion for the Drupal icon. You can feel free to remove it from your suggestion if you think it's best. #942

**Comment by LelouchFR** [comment]
Hi there @Jeesson @TomSkywalker , if you want them to get added, you can ask them at another repository, that is in rolling release and community driven, since the maintainer didn't showed up in months and these icons could be great [here](https://github.com/lelouchfr/skill-icons)

**Comment by TommyHyperTexter** [comment]
> Hi there @Jeesson @TomSkywalker , if you want them to get added, you can't ask them at another repository, that is in rolling release and community driven, since the maintainer didn't showed up in months and these icons could be great [here](https://github.com/lelouchfr/skill-icons)

Thanks, I will check it out.

**Comment by Jeesson** [comment]
Hello @LelouchFR, what's the point of doing that in your "fork"?
How to use the newly added icons?

**Comment by LelouchFR** [comment]
> Hello @LelouchFR, what's the point of doing that in your "fork"?
> 
> How to use the newly added icons?

Hi @Jeesso ,To use the new icons you just have to change your link from skill icons.dev to go-skill-icons.vercel.app/api

The point of doing that there, is that it is going to get added pretty much faster than here, but if you want to wait a decade long you can just go with it, since the last update on the repository here, is 2 months ago just the maintainer adding fundings.yml then just stopped looking at it, last real commit 3 months ago...

Since there is already the dreamweaver icon, to show you 

```md
![DreamWeaver](https://go-skill-icons.vercel.app/api/icons?i=dw)
```

![DreamWeaver](https://go-skill-icons.vercel.app/api/icons?i=dw)
